tests/baseclass.py

```diff
@@ -68,11 +68,11 @@ def setUp(self):
         self.temp_prefix = "temp-device-"
         (self.fd, self.path,) = tempfile.mkstemp(prefix=self.temp_prefix)
         self.f = os.fdopen(self.fd)
-        self.f.seek(140000)
+        self.f.seek(1400000)
```

Comment:
You have the same change twice, once here, once earlier in the file, and they are in two commits. Does this one perhaps belong to the next commit instead?

Reply:
It does indeed.